packages/thumbprint-react/components/Calendar/index.tsx

```diff
@@ -33,13 +16,16 @@ function throwError(message: string): void {
 }
 
 export function normaliseValue(value: CalendarProps['value']): Date[] {
-    if (value === null) {
+    if (value === null || !value) {
         return [];
     }
 
-    const valueArr: DateIsh[] = castArray<DateIsh>(value);
+    if (!isArray(value)) {
+        const valueArr: string[] = castArray<string>(value.toString());
```

Comment:
What's the reason we're converting the value to a string in this line, and then back to a date in the next line?

Reply:
Good catch! I think it's not needed!(I might played with it while fixing lint type issues). For non-array value, I can directly convert to array without type conversion to string and back to Date.

Updated!